src/test/java/org/prebid/server/auction/OrtbTypesResolverTest.java

```diff
@@ -36,44 +36,48 @@ public void normalizeTargetingShouldConvertArrayToFirstElementFieldForUserAndWri
         // given
         final JsonNode inputParam = mapper.createObjectNode().set("user",
                 mapper.createObjectNode().set("gender", array("male", "female")));
-        final List<String> errors = new ArrayList<>();
+        final PrebidLog prebidLog = PrebidLog.empty();
 
         // when
-        ortbTypesResolver.normalizeTargeting(inputParam, errors, "referer");
+        ortbTypesResolver.normalizeTargeting(inputParam, prebidLog, "referer");
 
         // then
         assertThat(inputParam).isEqualTo(mapper.createObjectNode().set("user",
                 mapper.createObjectNode().put("gender", "male")));
-        assertThat(errors).containsOnly("WARNING: Incorrect type for first party data field targeting.user.gender,"
-                + " expected is string, but was an array of strings. Converted to string by taking first element "
-                + "of array.");
+        assertThat(prebidLog.debug().getAllMessages()).hasSize(1)
```

Comment:
Redundant call to `hasSize`, just `containsExactly` is enough.
Also, please, check for similar occurences.